src/content/en/2021/jamstack.md

```diff
@@ -194,7 +205,9 @@ There's 7 most popular CDN providers for SSGs
 
 Jamstack CDN services are not just for network delivery. They provide a lot of functionality to allow developers to easy deploy and manage Jamstack sites. For example, Netlify provide easy to use functionality to deploy sites in scope of their service so developers can just update the code and the continuous deployment process is managed for them. <a hreflang="en" href="https://bejamas.io/compare/netlify-vs-vercel/">Jamstack CDNs provide many other features</a> such as _serverless functions_, _A/B testing_ etc.
   
-On the other hand, Cloudflare, Akamai, AWS are used not only for pure serving as CDN either, but can also provide protection service, DNS balancing etc. However, since we can't detect how exectly Cloudflare, Akamai, AWS are used, results could be false positives if we look at them as Jamstack enablers, since the "Jamstack" part could be handled on origin servers and so not actually on these services.
+On the other hand, Cloudflare, Akamai, AWS are not only used for pure serving as CDN either, but can also provide protection service, DNS balancing and more. However, since we can't detect how exactly Cloudflare, Akamai, and AWS are used, results could be false positives if we look at them as Jamstack enablers, since the "Jamstack" part could be handled on origin servers and so not actually on these services.
+
+{# TODO - can we divide that last sentence into 2 so it's not a run on? #}
```

Comment:
```suggestion
On the other hand, Cloudflare, Akamai, AWS are not only used purely for content deliver either, but can also provide protection service, DNS balancing and more. However, since we can't detect how exactly Cloudflare, Akamai, and AWS are used, results could be false positives if we look at them as Jamstack enablers. The "Jamstack" part could be handled on origin servers and so not actually on these services.
```